components/zoho_crm/actions/list-modules/list-modules.mjs

```diff
@@ -0,0 +1,17 @@
+import zohoCrm from "../../zoho_crm.app.mjs";
+
+export default {
+  key: "zoho_crm-list-modules",
+  name: "List Modules",
+  description: "Retrieves a list of all the modules available in your CRM account.",
+  version: "0.2.0",
```

Comment:
Since the key is changing, shouldn't this be version `0.0.1` of the "new" component?

Also remember to get the old key unpublished when this is merged